### PR TITLE
[Jetcaster] Add TwoPane from accompanist/adaptive for player UI

### PR DESCRIPTION
**Pull request by alexvanyo**
Using `TwoPane` from accompanist/adaptive to support postures and adaptive layout for the player screen in Jetcaster.

### CODE REVIEW

**Comment by riggaroo** [comment]
This can probably be a PR into `main` @alexvanyo as it doesn't seem to be an alpha or beta of compose that its depending on.

**Comment by alexvanyo** [comment]
@riggaroo accompanist/adaptive depends on Compose 1.3 for [`PlacementScope.coordinates`](https://developer.android.com/reference/kotlin/androidx/compose/ui/layout/Placeable.PlacementScope#coordinates()), and I think we don't want `main` to depend on 1.3 yet?

**Comment by riggaroo** [comment]
Ah cool, yeah thats fine, thanks!

File: Jetcaster/app/build.gradle.kts [hunk 1]
```diff
@@ -109,6 +110,7 @@ dependencies {
 
     implementation(libs.androidx.window)
 
+    implementation(libs.accompanist.adaptive)
```
Comment (Gurupreet) [inline Jetcaster/app/build.gradle.kts:113]: @alexvanyo So two pane will be part of accompanist adaptive?

Comment (alexvanyo) [reply Jetcaster/app/build.gradle.kts:113]: Yes, at least for the time being to explore its API shape and get feedback.

**Comment by alexvanyo** [comment]
@Gurupreet could you take another look here?

This is now targeted against main since Compose 1.3 is stable.

File: Owl/gradle/libs.versions.toml [hunk 2]
```diff
@@ -50,6 +50,7 @@ secrets = "2.0.1"
 targetSdk = "33"
 
 [libraries]
+accompanist-adaptive = { module = "com.google.accompanist:accompanist-adaptive", version.ref = "accompanist" }
```
Comment (Gurupreet) [inline Owl/gradle/libs.versions.toml:53]: @alexvanyo is there any reason it's added to all samples?

**Review by Gurupreet — APPROVED** [review]
Looks good! Can you add screenshot for future to visualise table top etc

File: Owl/gradle/libs.versions.toml (hunk 2 above)
Comment (riggaroo) [reply Owl/gradle/libs.versions.toml:53]: Yup - exactly that, it doesn't getting added to the actual import, just the toml file.

**Comment by alexvanyo** [comment]
@Gurupreet added an image and updated the README in the latest commit!

**Pull request merged**